### PR TITLE
Release 1.1

### PR DESCRIPTION
**Pull request by jonfoust**
**What this PR does / Why we need it**:
Release-1.1

### CODE REVIEW

**Comment by thisisnotapril** [comment]
@jonfoust we doing this?

**Comment by jonfoust** [comment]
Should we? The release requires at least one reviewer.

**Comment by sawagh** [comment]
Lets discuss this in the community meeting tomorrow. I am curious as to how the ongoing backfill feature work lines up with this (would prefer cutting 1.1 now so backfill feature has ample of time to bake in 1.2)

File: Makefile [hunk 1]
```diff
@@ -668,16 +668,11 @@ api/api.md: third_party/ build/toolchain/bin/protoc-gen-doc$(EXE_EXTENSION)
 	$(PROTOC) api/*.proto \
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
   		--doc_out=. \
-  		--doc_opt=markdown,api.md
+  		--doc_opt=markdown,api_temp.md
 # Crazy hack that insert hugo link reference to this API doc -)
```
Comment (calebatwd) [inline Makefile:672]: it seems like this may not be true anymore?

File: go.mod [hunk 2]
```diff
@@ -45,6 +45,7 @@ require (
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v1.2.1
+	github.com/pseudomuto/protoc-gen-doc v1.3.2 // indirect
```
Comment (aLekSer) [inline go.mod:48]: `go mod tidy` invocation removes this indirect dependency.

**Review by calebatwd — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (jonfoust) [reply go.mod]: I'll explore where this is used. I'll make sure if it is not part of the make release target it gets added in a future PR

File: Makefile (hunk 1 above)
Comment (jonfoust) [reply Makefile]: This does look like an additional step. will reproduce and be sure. I believe this change was made and merged in previous PR. If so, I'll reopen and make another change

Comment (jonfoust) [reply Makefile]: Reproduced and it seems to hold true.

File: go.mod (hunk 2 above)
Comment (jonfoust) [reply go.mod]: I will either add this to a future PR for either the Release process or make release target. Could not locate any use of it.

Comment (jonfoust) [reply go.mod]: Actually I found where it is used. 

https://github.com/googleforgames/open-match/blob/4b3147511bdcc292b711390532fd10ea6a5da8eb/Makefile#L696

Comment (jonfoust) [reply go.mod]: This looks to be used for generating documentation for the docs repository

Comment (aLekSer) [reply go.mod]: Seems that without go.mod changes should work with this make command:
https://github.com/googleforgames/open-match/blob/fcd590eca689311fa31b5a68c0935c2b11aa913a/Makefile#L561

Comment (jonfoust) [reply go.mod]: I see. I will submit a PR with the changes.

**Pull request merged**